### PR TITLE
Add build trigger url as env variable

### PR DESCRIPTION
**Pull request by fyangd**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did: Expose the url that triggered the build as an build environment variable.
- [x] Link to relevant issues in GitHub or Jira: https://github.com/jenkinsci/urltrigger-plugin/issues/30
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by fyangd** [comment]
@TonyNoble Please take a look

File: src/main/java/org/jenkinsci/plugins/urltrigger/URLTriggerCause.java [hunk 1]
```diff
@@ -9,9 +9,18 @@
 
     public static final String NAME = "URLTrigger";
     public static final String CAUSE = "A change within the response URL invocation";
+    public static String urlTrigger;
+
+    public static void SetUrlTrigger(String url) {
```
Comment (ellieMayVelasquez) [inline src/main/java/org/jenkinsci/plugins/urltrigger/URLTriggerCause.java:14]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (fyangd) [reply src/main/java/org/jenkinsci/plugins/urltrigger/URLTriggerCause.java]: thanks @ellieMayVelasquez!  I resolved your comment.

**Comment by fyangd** [comment]
Hi @TonyNoble please take a look.

**Comment by TonyNoble** [comment]
@fyangd - many thanks for this work!  I'll get this merged as soon as possible so we can get a release out 👍

**Pull request merged**